src/library/themes/createTheme.js

```diff
@@ -28,119 +26,115 @@ const spacing_3x = pxToEm(24);
 const spacing_4x = pxToEm(32);
 const spacing_8x = pxToEm(64);
 
-const grays = createColorRamp('gray', 'color_gray', color);
+const grayRamp = createColorRamp('gray', 'color_gray', color);
 
 export default function createTheme(
   baseColor?: Colors = 'blue',
   overrides?: { [string]: any } = {}
 ): { [string]: any } {
   /* Theme key naming convention:
    *
-   *   [property]_[target]_[variant]_[state]
+   *   [target]_[property]_[variant]_[state]
```

Comment:
I think this naming scheme change is great. If I'm trying to theme my h1s, all the available theme variables for them are next to each other in the docs, which is super helpful.